Header header
Obstacle[] obstacles           # 所有障碍物列表